boltzrpc/boltzrpc.proto

```diff
@@ -592,9 +607,10 @@ message LightningChannel {
 message SwapStats {
     uint64 total_fees = 1;
```

Comment:
I'd love for this to be much more granular in general. Just returning sums of stuff is not particularly useful when you consider it's 3 different swap types on 2 different chains. What actionable insight does that sum even give you in that context?

Reply:
> I'd love for this to be much more granular in general.

We could either return it grouped by swap type or allow for more filtering in the request, like we do with `ListSwaps`

> What actionable insight does that sum even give you in that context?

Just a general overview. For example, btcpay plugin users who use the client to receive lightning payments are probably interested in how much fees they are paying compared to the total volume.